### PR TITLE
Highlightjs - closes #29

### LINKED ISSUE #29: Replace prettify with Highlight.js?
This is really nice: https://highlightjs.org/

And it supports the SCSS syntax very well:

screen shot 2015-02-21 at 19 27 45

Thoughts?

### PR DESCRIPTION
**Pull request by kaelig**
- Replaced Prettify with [Highlight.js](https://highlightjs.org/)
- Removes the need for an external script to highlight code

### CODE REVIEW

**Comment by kaelig** [comment]
Looks like highlightjs works well. I suggest we merge this and ship v4.0.0

File: demos/demo.html [hunk 1]
```diff
@@ -181,18 +171,12 @@ <h2 id="code">Source code and syntax</h2>
 
 <h3>Syntax highlighting</h3>
 
-<p>To trigger syntax highlighting using <a href="https://code.google.com/p/google-code-prettify/">prettify</a>, prefix the <code>&lt;pre&gt;</code> with a prettify inline hint such as <code>&lt;?prettify linenums=1?&gt;</code>, or add the <code>prettify</code> class, as documented in the <a href="https://code.google.com/p/google-code-prettify/wiki/GettingStarted">prettify wifi</a>.  Since prettify auto-loads all its lang files, we use it from the Cloudflare CDN.  Put this in your page to enable support:</p>
-
-<?prettify linenums=1?>
-<pre><code>&lt;script src="//cdnjs.cloudflare.com/ajax/libs/prettify/r298/prettify.js">&lt;/script>
-&lt;script&gt;addEventListener('load', function(event) { prettyPrint(); }, false);&lt;/script&gt;</code></pre>
-
+<p>o-techdocs brings syntax highlighting out of the box using <a href="https://highlightjs.org">Highlight.js</a>.</p>
```
Comment (triblondon) [inline demos/demo.html]: Need to document how to opt-out, and how to specify language.

File: src/js/gist-it.js [hunk 2]
```diff
@@ -19,18 +20,14 @@ $(function() {
...
+			content = content.replace(/<\/pre>$/, "</code></pre>");
 
 			el.innerHTML = content;
 			window[callbackName] = undefined;
 
-			// If available, re-run prettify so that the new content is highlighted
-			if ("prettyPrint" in window) {
-				window.prettyPrint();
-			}
+			// If available, re-run highlighter so that the new content is highlighted
```
Comment (triblondon) [inline src/js/gist-it.js]: You're doing this unconditionally, rather than only if available

Comment (kaelig) [inline src/js/gist-it.js]: Good catch, the comment is badly phrased

File: demos/demo.html (hunk 1 above)
Comment (kaelig) [inline demos/demo.html]: Ha, it's in by default. I suggest we build an opt-out mechanism when someone requests it. What do you think?

File: src/js/highlight.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
+/*global require*/
+/**
+ * Add a second navigation menu to quickly navigate to
+ * anchors in the page.
```
Comment (AlbertoElias) [inline src/js/highlight.js]: Is this comment relevant?

Comment (kaelig) [inline src/js/highlight.js]: Woops, bad copy-paste

**Comment by AlbertoElias** [comment]
:+1:

**Pull request merged**